### ISSUE TITLE
Few questions about this repo by 🇷🇺

### ISSUE THREAD

**Issue by AdamBrianBright** [vshymanskyy/StandWithUkraine#2]
I'm against the war, but I'm just a civilian, working on gaming related projects, so seeing this repo made me curious about following questions:

1. What makes provided sources trusted and how they gather and validate information in the era of fakes and day-to-day misinformation even in peaceful time?
2. What is your motivation for encouraging people to stop using Russian software?
3. Do you plan to provide the full list of projects, technologies and products, created or maintained by Russian people? 
4. Do you suggest everyone else to stop using nginx, postgresql, google, etc. because Russian leaders decided to start all this?
5. How do you distinct average Russian people and those, responsible for current situation?
6. Do you also suggest to stop any interaction with any Russian living in different country?

**Comment by vshymanskyy** [comment]
Hi. @AdamBrianBright thanks for reaching me out.

Many of your questions are related to my personal point of view. 

Regarding my personal opinion.

1. I'm not **forcing** anyone to take actions based on information in this repo. I will however, do my best to take into account and issues, PRs and comments. Russian input included.
2. Kharkiv, Kyiv, Bucha, Irpin (my native town), Sumy, Kherson, ... Please google.
3. Yes, I will do my best to provide such lists
4. Your army and government is **your moral responsibility**.
5. Your army and government is **your moral responsibility**.
6. Absolutely **not**.

**Comment by vshymanskyy** [comment]
@AdamBrianBright please point to the specific questions that were not answered. I'll try to elaborate

**Comment by AdamBrianBright** [comment]
1. I was curious about trustee of sources, how can we know that everything there is true? 
4, 5. These questions were answered as it's all our* fault regardless. As a citizen of Russia it is at least not an answer for me. 

I don't want to escalate hatred in such lovely place as GitHub, so if you feel like it should be done, please, close (or remove? idk) issue. Thank you

**Comment by vshymanskyy** [comment]
Regarding point n.6, I'll try to update the README.md to express this.

**Comment by vshymanskyy** [comment]
Sorry, I don't see any hate speech here. The discussion is productive, I think.

**Comment by vshymanskyy** [comment]
4. Long story short, Yes. Unless Russia stops invasion, withdraws troops.
5. Russian people did allow this. They do fund the Russian Army. They did elect Putin, didn't they?

**Comment by vshymanskyy** [comment]
Regarding question n.1, I'm not sure how to answer.
- Regarding war in general: My own home town is burning. I did see Russian Army invading Ukraine with my naked eyes.
- Regarding sources: as you can see there is just a few of them. I selected them because none of them is backed by any governments, funds, tycoons, etc.

**Comment by vshymanskyy** [comment]
@AdamBrianBright I hope all questions were answered to an acceptable level.

**Comment by AdamBrianBright** [comment]
1. Understandable, I'm sorry to hear that. 
4. Understandable, thank you for the answer.
5. We can discuss about the following to 5th point questions a lot, but imho:

We didn't allow it, we were never asked. We don't fund army, we pay taxes (because it's law, and we don't want to be in jail), which goes to federal fund, which gains money from different sources (mostly gas/oil). I don't know if we elected Putin, it's a mystery for me. And we didn't ask to be born and raised in Russia. Especially we didn't go to work with thoughts like _"I wish my tax money will fund invasion"_. So... it's complicated _I guess_? And there are a lot of people protesting and being arrested for it, so it doesn't seem like everyone want war. And I don't _believe*_ like loosing a bit of money from taxes will run those money-packed pockets dry.

**Comment by vshymanskyy** [comment]
This is my residential area. Many civilians have died here.
![image](https://user-images.githubusercontent.com/1506708/156559849-b99ad578-9d2f-4246-aa85-e07185990d81.png)

Now, I'm not calling for annihilating of Russian people. I just want them to stop paying for this war.
As long as dozens of civilians die in Ukraine from Russian missiles, your arguments do not hold water.
Please read the history on nazi and post-nazi Germany.

**Comment by AdamBrianBright** [comment]
I understand the tragedy and hope it will end, but this conversation seems went to emotions, so I would call it an end.

**Comment by vshymanskyy** [comment]
Reopening for further reference. Discussion closed

**Comment by Py-debug** [comment]
But please explain to this video，fucking Nazi！
https://video.weibo.com/show?fid=1034:4742655347982440&amp;mark_id=999_reallog_mark_ad%253A999%257CWeiboADNatural
I'm also against the war, but I can't stand the Nazi, so I'm for Russia, and please say 
"u are against the wat and death" to the citizen that killed by Ukraine army

**Comment by Py-debug** [comment]
when i share the video in twitter, i was forbidden
Is this freedom of speech?
Is this the Ukrainian government you support?

**Comment by webknjaz** [comment]
@Py-debug please stop posting russian fascist fakes. Thanks in advance.

**Comment by Py-debug** [comment]
Thank you for showing me the so-called freedom of speech,
If you think what I posted is false, please show evidence
Your deletion will only make me more convinced that you are hiding something @webknjaz

**Comment by vshymanskyy** [comment]
@Py-debug you have enough evidence in this thread. And in Google search bar.
Sorry, this is not discussable. Should I sponsor you a visit to Ukraine?

**Comment by Py-debug** [comment]
Ok, Should I sponsor you a visit to Odessa?Donetsk?Lugantsk?
If I doubt something, do I have to do it on the ground?
You at least have internet, electricity here, and those tens of thousands of victims in Donetsk, Lugentsk, Odessa, who fell under the guns of their own government, fell under the guns of the Nazis, and are still forbidden to tell what happened to them.
you means, What is against you is not discussable?
Grateful to github, at least I can say these things.

- The Odessa tragedy, also known as the Odessa bloodbath, which left 46 people dead and 214 injured, was a massacre of peaceful demonstrators protesting against the Nazis, supported, planned and led by the right-wing Ukrainian neo-Nazis, who forced peaceful elderly people, pregnant women and children into buildings and set them on fire, while those who tried to escape were strangled to death. The Ukrainian government informed that it was a fabrication of rumors and shifted the responsibility to the victims.
- 
- Russia's Investigative Committee Cloud on Wednesday launched a criminal case after a spate of civilian mass graves were found in the Donbass region of eastern Ukraine. The agency issued a statement saying, "A criminal case was opened as a result of the discovery of civilian mass graves in the Donbass region. It is based on Part 1 of Article 356 of the Russian Criminal Code - mistreatment of civilians, use of means and methods prohibited by international treaties in the context of armed conflict." The statement said, "In 2014, the bodies of at least 295 civilians who died as a result of indiscriminate shelling by the Ukrainian armed forces were exhumed. It has been learned that among the remains there were female bodies of different ages."
- The young hero in the picture is named Vadim Papula, a native of Odessa, born on July 24, 1996, today is his birthday. He is honest, kind and tolerant. At the age of 16, he joined the Ukrainian Komsomol and became a delegate to the Congress of the Komsomol in Kiev in the same year. The situation in Odessa deteriorated as a result of the Russian-Ukrainian crisis, and Ukrainian nationalists planned to occupy Odessa regional government institutions. Starting with pro-Russian actions, he was involved in a number of activities such as opposition to violence in Kiev. on May 2, 2014, Vadim participated as a member of the Odessa squad in a clash with Ukrainian nationalists on the Odessa factory peyecKon square. When news came that the Ukrainian nationalists were advancing in the direction of their camp, Vadim and his comrades were forced to retreat to the trade union building. His mother remembers the moment when Vadim left the house - he dressed quickly and left, perhaps having received a phone call or seen a message on social networks. The last time his grandmother saw him, she asked him, "Where are you going?" To which he replied: "I'm going to protect you". Together with his comrades, he was forced to hide in the trade union building, among a crowd of fanatical nationalists. After jumping out of a burning building during a fire in the Odessa trade union building, in which he hid the red flag on his chest, he was caught, brutally mutilated and beaten to death by the nationalists.In October 2014, young Marxists in Lipetsk, Russia, announced a fundraising effort for a monument to Vadim in Odessa. Vadim Papula also became one of the central images in French journalist Paul Moreira's documentary "Ukraine, the Mask of Revolution.
![image](https://user-images.githubusercontent.com/30771407/156725556-ca805a03-bdae-4751-912a-4599cd82948e.png)

**Comment by Py-debug** [comment]
If you ban me, it will be my honor

**Comment by nevcos** [comment]
@AdamBrianBright It's definitely a bad time to be Russian, but it's a terrible time to be Ukrainian. 
From one day to the other they lost everything that you give for granted, like having peace, home, food, family and life.

Honestly I think everyone can and should help, and I think Russian people are the first ones that can make the difference.
If Ukrainian people are willing to die for protecting their country and their rights, I really beg that Russian people go all outside and start protesting in masses for this nonsense war. **Nothing is more powerful then people** and it's simply impossible for the government to arrest everyone.

Please fight for what you believe, fight for letting know the truth, fight for having a fair country with a true democracy, fight for the human rights!
Everyone... together we can make the difference 💪

**Comment by internalsystemerror** [comment]
Note: You won't be able to guess my political views, so don't even try :D.

I decided to investigate the claims made by @Py-debug and it seems there is plenty of information about it. Those times of the revolution in Ukraine were certainly bloody indeed, especially what happened in Odessa. I will let others investigate for themselves but I would say not to take any evidence on its own, but in the whole context of what was occurring in Ukraine at that time.

Who is in power in Ukraine, what the laws are in Ukraine, these are problems for Ukraine and no one else. That seems fairly self evident.

There is censorship of vital information in Russia right now, preventing the population from being able to investigate and understand for themselves. As I understand it, this repository is an effort to help inform the Russian population that this is happening.

How to get accurate information, fact check etc? Sources, many of them. Take into context whether they have a reason to mislead, are there any examples of them misleading in the past (evidence, not rumour or conjecture)? Look to overseas sources from those unconnected to either side in this conflict, what are they reporting to their people?

But when Rupert Murdoch's global news empire, the BBC and Al Jazeera are all saying the same thing, that's a fairly strong indication it may be true.

And if you want to know how the people are responsible for their government and what they can do, I suggest reading up on Ukraine 2014.

**Comment by vshymanskyy** [comment]
@internalsystemerror fair enough.

**Comment by Py-debug** [comment]
@internalsystemerror 
I'm sorry, I was watching videos that were too gory to videos that there could still be massacres today.
Actually, I am not originally from Ukraine and have no right to say anything. It's just that I think the BBC and CNN videos are full of bias and falsehoods, and similar content would be deleted immediately on twitter. The control over the news itself is part of the war. The superior side will win the opinion war with huge amount of information, but the truth often does not shift because of superiority.
I decided it was better not to do anything, and everything will come out after the war is over.
Sorry for the previous presumptuous behavior.

**Comment by Py-debug** [comment]
@internalsystemerror
If you are interested in this history, I am very happy to be able to share with you all the videos I have saved, including the ones that are scorned in mainstream media such as twitter and facebook. False evidence is always false, and I am sure you will be able to tell with your own ability.

**Comment by internalsystemerror** [comment]
@Py-debug Interesting that twitter and facebook are considered mainstream media :D. Given anyone can post anything on these platforms I consider social media the most untrustworthy and most worthy of scrutiny. You don't know the source; you don't know whether it really is where and when it suggests; I'm not a military expert able to spot a vehicle and know exactly what it is, what its for, who's it is, whether its stolen etc etc.

I would say more, but I don't want to derail this from the topic of the restriction of accurate information in Russia and by Russian news sources which is the purpose of this repository. The assumption being, and one that I share, that if the Russian people were given all the facts, then things may change (to put it politely).

**Comment by Einenlum** [comment]
Just FYI, I see that JetBrains is in the list of Russian products, but it [officially condemned the war in Ukraine](https://twitter.com/jetbrains/status/1496786254494670851).

**Comment by kstupak** [comment]
@AdamBrianBright To any russian citizen that is "againstthewar" and "werenotallthesame" I say this - you retaining russian passport and citizenship means your consent to and shared responsibility for actions of your elected government. So unless you're actively prohibiting your govt from committing unlawful acts or unless you're denouncing russian citizenship - you are an accomplice to all of the war crimes and are equally responsible for violence, destruction, and suffering caused by russian govt anywhere in the world. 
Anticipating your exclamations - this situation is binary and there's no grey area for you to hide in.

**Comment by TheMikeyRoss** [comment]
> @AdamBrianBright To any russian citizen that is "againstthewar" and "werenotallthesame" I say this - you retaining russian passport and citizenship means your consent to and shared responsibility for actions of your elected government. So unless you're actively prohibiting your govt from committing unlawful acts or unless you're denouncing russian citizenship - you are an accomplice to all of the war crimes and are equally responsible for violence, destruction, and suffering caused by russian govt anywhere in the world. Anticipating your exclamations - this situation is binary and there's no grey area for you to hide in.

Let's use that same logic with America and Israel.

**Comment by Py-debug** [comment]
@internalsystemerror Because I suddenly think there is no point in playing to the bull, I live in a happy and stable country and I will always support my country. So I don't want to meddle in this matter anymore, just wait for the final result. I believe the Nazi war mongers will be judged.

**Comment by internalsystemerror** [comment]
@kstupak I agree, given all the facts this is one of the most obvious examples of right and wrong that I've seen in my life time. But based on the above results, I'm not sure if they're purposely trying to spread disinformation, or just refuse to investigate the facts for themselves. Either way, I fear for humanity.

**Comment by vshymanskyy** [comment]
Sorry, I had to clean up the thread. With @Py-debug and @AdamBrianBright  removing their comments randomly, it was really hard to follow. I need to restrict it again. Hope for understanding.
